src/category/monad/cont.lean

```diff
@@ -17,13 +18,12 @@ structure monad_cont.label (α : Type w) (m : Type u → Type v) (β : Type u) :
 
 def monad_cont.goto {α β} {m : Type u → Type v} (f : monad_cont.label α m β) (x : α) := f.apply x
 
-class monad_cont (m : Type u → Type v)
-extends monad m :=
+class monad_cont (m : Type u → Type v) :=
```

Comment:
Why did you remove `extends monad m` here?

Reply:
Because `monad` contains data and if `monad_state`, `monad_reader` and `monad_writer` each extend `monad`, it won't be clear which monad we're talking about if we assume that a variable `m` has both `monad_state` and `monad_cont`.